wikimedia/wikipedia-ios
# Move function to NSMutableAttributedString extension

**Phabricator:** N/A

### Notes
* Addressing dome feedback from another PR https://github.com/wikimedia/wikipedia-ios/pull/4321, we're transforming this method into a class extension that can be reused

### Test Steps
1.  Go to a talk page that has a coffee roll, make sure that the text aligns to the top of the view
